<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_4">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>117</width>
        <height>157</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-56</y>
         <width>115</width>
         <height>190</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>91</width>
        <height>151</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <attribute name="label">
        <string>朋友</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <attribute name="label">
        <string>黑名单</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <attribute name="label">
        <string>家人</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>321</width>
        <height>211</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>谷歌</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>百度</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>传智</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
   </widget>
   <widget class="QPushButton" name="btn_scrollArea">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ScrocllArea</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ToolBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ToolBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TabWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tab_Widget</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选中兰博基尼</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>104</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>104</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>570</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>540</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>630</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/image/la.jpg</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
